snippets/csharp/System.Net.Http/HttpCompletionOption/HttpCompletionOptionSnippets.csproj

```
@@ -0,0 +1,10 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net8.0</TargetFramework>

```

```suggestion
    <TargetFramework>net9.0</TargetFramework>
```
Bump sample to .NET 9 (using new `LoadIntoBufferAsync` overload)